### ISSUE TITLE
Invalid request if `timeUrl` contains a query parameter

### ISSUE THREAD

**Issue by jkmatila** [TTLabs/EvaporateJS#444]
EvaporateJS uses an invalid URL for fetching the time if the `timeUrl` already contains query parameters. It will just append the request time at the end of the URL separated by a `?` even if 
query parameters already exist. See here: [evaporate.js:152](https://github.com/TTLabs/EvaporateJS/blob/master/evaporate.js#L152).

Steps to reproduce: 

Set timeUrl to `http://example.com/time?anotherparam=foo`

Expected:

EvaporateJS should get `http://example.com/time?anotherparam=foo&requestTime=...`

Observed:

EvaporateJS gets `http://example.com/time?anotherparam=foo?requestTime=...`

**Comment by jakubzitny** [comment]
Yeah this looks incorrect, PR with a fix would be very appreciated 😇
